openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -241,6 +246,18 @@ private void getTargetOutputPath() {
             }
         }
     }
+    private void setLicenseHeader() {
+        String licenseHeaderFileName = "";
+        if (licensePath && argList.size() > 1 && argList.get(1).endsWith(".txt")) {
```

Comment:
If we set licensePath as String type we can directly use its value to the call `setLicenseHeaderFilePath` function if its value is not null. :) if there is any possibility to change shall we do it?

Reply:
Updated